### DISCUSSION TITLE
[svelte]: Dev Tools

### DISCUSSION THREAD

**Discussion by SaintPepsi** [TanStack/query#7460] [General]
At my work we use Tanstack Query Devtools, which is an amazing way to quickly debug your queries. 
Svelte Query doesn't have this implementation yet. 

I would like to contribute and get the same tool working for svelte. 

What will be required to implement this for svelte? 
Do I just create a pull request and start? 
Do we need to make sure the styling matches?

Kind Regards
- Ian

**Comment by SaintPepsi** [comment]
I just found that they already exist, but it was only because I looked in the discord discussions:
https://discord.com/channels/719702312431386674/1117372337243816027

`@tanstack/svelte-query-devtools` How would I add this to the tanstack docs?

**Comment by mindmind** [comment]
https://github.com/TanStack/query/pull/8149 FYI
